pkgs/applications/window-managers/awesome/default.nix

```diff
@@ -4,28 +4,23 @@
 , compton, procps, iproute, coreutils, curl, alsaUtils, findutils, xterm
 , which, dbus, nettools, git, asciidoc, doxygen
 , xmlto, docbook_xml_dtd_45, docbook_xsl, findXMLCatalogs
-, libxkbcommon, xcbutilxrm
+, libxkbcommon, xcbutilxrm, hicolor_icon_theme,
+#Hicolor is used for 4.0 themes such as CopyCat's awesome theme collection
+enableHicolorTheme ? false
+
 }:
 
-let
-  version = "4.0";
-in with luaPackages;
+assert enableHicolorTheme -> hicolor_icon_theme !=null;
```

Comment:
You don't actually _use_ `hicolor_icon_theme` anywhere (or I'm just very sleepy but browser search seems to agree with me).